include/lbgrid.h

```diff
@@ -3,18 +3,20 @@
 #include <iostream>
 
 class lbgrid {
-private:
+ private:
   int nx_;
   int ny_;
   const int Q = 9;
-  double ***f = new double **[nx_];
+  double*** f = new double**[nx_];
   double total_mass = 0.0;
 
-public:
+ public:
   lbgrid(int nx, int ny);
+  const int nx_value();
+  const int ny_value();
   void initialize_density(double rho);
-  void print_f();
-  void check_density();
+  const double density_function(int i, int j, int k);
```

Comment:
const will be at the end of the function, this will only return a const, we are explicitly saying nothing chagnes inside the `density_function`